### PR TITLE
Em 322 redundant prs

### PR DESCRIPTION
**Pull request by esparratacus**
## What does this PR do?

adds a relation between `CompletedReviewTurnaround` and `PullResquest` models
adds a relation between `ReviewTurnaround` and `PullResquest` models
Added two processors to populate the foreign keys on existing data which will be called over two rake tasks

Creates two rake tasks to populate the aforementioned tables with the corresponding foreign keys

## After deploy

run the following commands (rake tasks):
`rake one_time:set_pull_request_to_review_turnaround`
`rake one_time:set_pull_request_to_completed_review_turnaround`

Resolves EM-322

### CODE REVIEW

File: spec/services/processors/completed_review_turnaround_pull_request_backfiller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe Processors::CompletedReviewTurnaroundPullRequestBackfiller do
+  describe '.call' do
+
+    let(:department) { Department.find_by(name: 'backend') }
```
Comment (santiagovidal) [inline spec/services/processors/completed_review_turnaround_pull_request_backfiller_spec.rb:6]: I don't think you need this

File: spec/services/processors/review_turnaround_pull_request_backfiller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe Processors::ReviewTurnaroundPullRequestBackfiller do
+  describe '.call' do
+
+    let(:department) { Department.find_by(name: 'backend') }
```
Comment (santiagovidal) [inline spec/services/processors/review_turnaround_pull_request_backfiller_spec.rb:6]: Same here

**Review by santiagovidal — APPROVED** [review]
Minor comments, looks good otherwise 👍

**Review by brunoparma88 — COMMENTED** [review]
I think that **pull_request_id** shouldn't  not accept null values

File: spec/services/processors/review_turnaround_pull_request_backfiller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    let(:ruby_lang) { Language.find_by(name: 'ruby') }
+    let!(:repository) { create(:repository, language: ruby_lang) }
+    let!(:review_request) { create(:review_request, repository: repository) }
+    let(:review_turnaround) do
+      build(:review_turnaround,
+            review_request: review_request,
+            pull_request: nil)
+    end
+
+    before { review_turnaround.save!(validate: false) }
```
Comment (sebastiancaraballo) [inline spec/services/processors/review_turnaround_pull_request_backfiller_spec.rb:14]:
```suggestion
    before { review_turnaround.save!(validate: false) }
    
```

**Review by sebastiancaraballo — APPROVED** [review]

**Comment by sebastiancaraballo** [comment]
Remember to add `not_null` constraint on a different PR 😄

**Pull request merged**